packages/react/src/ReactCurrentBatchConfig.js

```diff
@@ -8,16 +8,27 @@
  */
 
 import type {BatchConfigTransition} from 'react-reconciler/src/ReactFiberTracingMarkerComponent';
+import type {
+  EventPriority,
+  NoEventPriorityType,
+} from 'react-reconciler/src/ReactEventPriorities';
 
 type BatchConfig = {
   transition: BatchConfigTransition | null,
+  eventPriority: EventPriority,
 };
 /**
  * Keeps track of the current batch's configuration such as how long an update
  * should suspend for if it needs to.
  */
 const ReactCurrentBatchConfig: BatchConfig = {
   transition: null,
```

Comment:
This will eventually have to move to AsyncContext where as the eventPriority doesn't.